test/Directory.Build.props

```diff
@@ -1,7 +1,11 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
-  <PropertyGroup>
-    <IsTestProject>true</IsTestProject>
-  </PropertyGroup>
+  <Choose>
+    <When Condition="$(MSBuildProjectName)!='coverlet.tests.projectsample.fsharp'">
+      <PropertyGroup>
+        <IsTestProject>true</IsTestProject>
```

Comment:
Can you force false directly on new project `coverlet.tests.projectsample.fsharp.fsproj `?
Or in future we need to add every time projects here.

Reply:
Yes sorry didn't think about this. I was getting a error when ```dotnet test``` the fsharp project and I didn't know why.
![image](https://user-images.githubusercontent.com/19378678/120052535-1182b100-c026-11eb-82eb-61d2143dfac4.png)
Took me a good amount of time to figure out it is the ```Directory.Build.pros```. The other c# class library projects don't have this issue even when ```<IsTestProject>true</IsTestProject>```.